docs/ref/models/fields.txt

```diff
@@ -2439,7 +2439,8 @@ Field API reference
     .. method:: formfield(form_class=None, choices_form_class=None, **kwargs)
 
         Returns the default :class:`django.forms.Field` of this field for
-        :class:`~django.forms.ModelForm`.
+        :class:`~django.forms.ModelForm`. If ``None`` is returned, this field
+        is excluded from the :class:`~django.forms.ModelForm`.
```

Comment:
```suggestion
        :class:`~django.forms.ModelForm`.

        If :meth:`~Field.formfield` is overridden to return ``None``, this field
        is excluded from the :class:`~django.forms.ModelForm`.
```
`None` will not be returned by the default implementation (which you can see when clicking the "source" link), so I think it's worth us emphasizing that it's been overridden